gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/Diagnostics.java

```diff
@@ -16,112 +16,42 @@
 
 package com.palantir.gradle.dist.service;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonValue;
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.palantir.gradle.dist.tasks.CreateManifestTask;
-import com.palantir.logsafe.Preconditions;
-import com.palantir.logsafe.SafeArg;
-import com.palantir.logsafe.exceptions.SafeIllegalArgumentException;
 import java.io.File;
 import java.io.IOException;
-import java.io.Serializable;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
-import java.util.Objects;
-import java.util.regex.Pattern;
 import org.gradle.api.GradleException;
 import org.gradle.api.Project;
-import org.immutables.value.Value;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public final class Diagnostics {
     private static final Logger log = LoggerFactory.getLogger(Diagnostics.class);
+    private static final String EXAMPLE =
+            "[{\"type\":\"foo.v1\", \"docs\":\"...\"}, \"{\"type\":\"bar.v1\", " + "\"docs\":\"...\"}]";
 
-    // This is the format the sls-spec wants list items to be. <code>{ type: "foo.v1", docs: "Lorem ipsum" }</code>.
-    @Value.Immutable
-    @JsonDeserialize(as = ImmutableSupportedDiagnostic.class)
-    public interface SupportedDiagnostic extends Serializable {
-        String EXAMPLE = "[{\"type\":\"foo.v1\", \"docs\":\"...\"}, \"{\"type\":\"bar.v1\", \"docs\":\"...\"}]";
-
-        DiagnosticType type();
-
-        String docs();
-    }
-
-    public static List<SupportedDiagnostic> parse(Project proj, File file) {
+    public static List<ObjectNode> parse(Project proj, File file) {
         Path relativePath = proj.getRootDir().toPath().relativize(file.toPath());
         String string = null;
         try {
             string = new String(Files.readAllBytes(file.toPath()), StandardCharsets.UTF_8).trim();
-            List<SupportedDiagnostic> value =
-                    CreateManifestTask.jsonMapper.readValue(string, new TypeReference<List<SupportedDiagnostic>>() {});
+            List<ObjectNode> value =
```

Comment:
`Map<String, Object>`?

Reply:
eh I actually prefer the ol ObjectNode types, it feels a bit more 'pure data' than having that `Object` value type, as technically that could be one of those jackson vulns like a FileOutputStream (which is technically an object)